assets/src/stories-editor/blocks/amp-story-page-attachment/edit.js

```diff
@@ -81,42 +59,39 @@ class PageAttachmentEdit extends Component {
 			color: textColor.color,
 			backgroundColor: appliedBackgroundColor,
 		};
+
 		return {
 			style: attachmentStyle,
 			attachmentClass,
 		};
-	}
+	};
 
-	render() {
-		const {
-			attributes,
-			setAttributes,
-		} = this.props;
+	useEffect( () => {
+		const { style, attachmentClass } = getWrapperAttributes();
+		const newAttributes = { wrapperStyle: style };
+		newAttributes.attachmentClass = attachmentClass;
+		setAttributes( newAttributes );
+	}, [ backgroundColor, customBackgroundColor, textColor ] );
 
-		const {
-			openText,
-		} = attributes;
-
-		return (
-			<>
-				{ this.state.isOpen &&
-					<AttachmentContent
-						setAttributes={ setAttributes }
-						attributes={ attributes }
-						toggleAttachment={ this.toggleAttachment }
-					/>
-				}
-				{ ! this.state.isOpen &&
-					<AttachmentOpener
-						setAttributes={ setAttributes }
-						toggleAttachment={ this.toggleAttachment }
-						openText={ openText }
-					/>
-				}
-			</>
-		);
-	}
-}
+	return (
+		<>
+			{ isOpen &&
```

Comment:
Convert this to a ternary - that will also get rid of the fragment and reduce the complexity of the return.